### ISSUE TITLE
Order book requests are not working

### ISSUE THREAD

**Issue by tistis96** [Soneso/stellar-php-sdk#2]
I have this code:
```
<?php
require __DIR__ . '/vendor/autoload.php';

use Soneso\StellarSDK\StellarSDK;
use Soneso\StellarSDK\Asset;

$sdk = StellarSDK::getPublicNetInstance();

$sellingAsset = Asset::createNonNativeAsset('yXLM', 'GARDNV3Q7YGT4AKSDF25LT32YSCCW4EV22Y2TV3I2PU2MMXJTEDL5T55');
$orderBook = $sdk->orderBook()->forBuyingAsset(Asset::native())->forSellingAsset($sellingAsset)->limit(1)->execute();
$offerAmount = $orderBook->getBids()->toArray()[0]->getAmount();
$offerPrice= $orderBook->getBids()->toArray()[0]->getPrice();
```
When I try to send a request to order book, I encounter this error:

> PHP Fatal error:  Uncaught Error: Typed property Soneso\StellarSDK\Responses\OrderBook\OrderBookResponse::$bids must not be accessed before initialization in C:\Users\USER\Desktop\projects\vendor\soneso\stellar-php-sdk\Soneso\StellarSDK\Responses\OrderBook\OrderBookResponse.php:50

The problem persist on getBids() method. I also ran the test from StellarSDKTests/TradesTest.php and it passes succesfully.

I'm using:
- windows 10;
- php version 8.1.1

**Comment by christian-rogobete** [comment]
Hello @tistis96,
thank you for reporting this issue. I fixed it in this commit :https://github.com/Soneso/stellar-php-sdk/commit/0f7edb10ce339a9cfb821a820e985e02ceab4e4e
it is published in [release 0.4.0](https://github.com/Soneso/stellar-php-sdk/releases/tag/0.4.0)
pls. update your sdk version and try again.

**Comment by danbradster** [comment]
That same code crashes with error 500 for me.  I have my php file in the directory above /vendor/.



<?php
require __DIR__ . '/vendor/autoload.php';
//or this
//require 'vendor/autoload.php';

use Soneso\StellarSDK\StellarSDK;
use Soneso\StellarSDK\Asset;

$sdk = StellarSDK::getPublicNetInstance();

$sellingAsset = Asset::createNonNativeAsset('yXLM', 'GARDNV3Q7YGT4AKSDF25LT32YSCCW4EV22Y2TV3I2PU2MMXJTEDL5T55');
$orderBook = $sdk->orderBook()->forBuyingAsset(Asset::native())->forSellingAsset($sellingAsset)->limit(1)->execute();
$offerAmount = $orderBook->getBids()->toArray()[0]->getAmount();
$offerPrice= $orderBook->getBids()->toArray()[0]->getPrice();
